roles/postfix/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+# tasks file for roles/postfix
+- name: Postfix | Install Postfix
+  ansible.builtin.apt:
+    name: postfix
+    state: present
+    update_cache: true
+
+- name: Postfix | Configure Postfix as relay host
+  ansible.builtin.lineinfile:
```

Comment:
With `lineinfile` I think the task would just add the line, so if the file already had `smtp_use_tls = no`, we might end up with both lines in the config file. Something to test.